heareval/tasks/pipeline.py

```diff
@@ -171,7 +171,9 @@ def slugify_file_name(relative_path: str):
 
         The basic version here takes the filename and slugifies it.
         """
-        return f"{slugify(str(Path(relative_path).stem))}"
+        slug_text = str(Path(relative_path).stem)
+        slug_text = slug_text.replace("-", "_negative_")
+        return f"{slugify(slug_text)}"
```

Comment:
speech_commands overwrites slugify_file_name. That is really risky for a bug.

Either:
1) speech commands does not override and it uses the same
or
2) the generic class method has slugify and is a *wrapper* method that fixes the - in the  relpath and calls a second method that does slugify on the wrapper fixed relpath, which speech commands also uses.

Reply:
dcase is a special case here using "-" to refer to a negative, this method should stay the same and dcase should override slugify_file_name with the changes above.

Reply:
Yes. We can override for `dcase` slugify to make it more explicit, and explainable to why we are doing this. Because other tasks(in general) might have `-` in their name, and we might not want to convert that to `-`. Thanks for the suggestion. Will change